src/locales/translations/ptbr.ts

```diff
@@ -0,0 +1,79 @@
+export default {
+  favorites: "Favoritos",
+  internet: "Internet",
+  partners: "Parceiros",
+  recently_joined: "Entrado Recentemente",
+  nickname: "Apelido",
+  settings: "Configurações",
+  minimize: "Minimizar",
+  maximize: "Maximizar",
+  close: "Fechar",
+  add_server_modal_description_1: "Adicione servidores manualmente à lista de favoritos.",
+  add_server_modal_description_2: "Exemplo: 127.0.0.1:7777",
+  add: "Adicionar",
+  server: "Servidores",
+  address: "Endereço",
+  players: "Jogadores",
+  server_join_prompt_enter_password:
+    "Este servidor está trancado, por favor digite a senha.",
+  server_join_prompt_enter_password_input_placeholder: "Digite a senha...",
+  server_join_prompt_nickname_input_placeholder: "Digite o Apelido...",
+  connect: "Conectar",
+  copy: "Copiar",
+  remove_from_favorites: "Remover dos favoritos",
+  add_to_favorites: "Adicionar aos favoritos",
+  filters: "Filtros",
+  filter_only_omp_servers: "Somente servidores open.mp",
+  filter_non_empty_servers: "Servidores não vazios",
+  filter_unpassworded_servers: "Servidores sem senha",
+  rule: "Regra",
+  value: "Valor",
+  player: "Player",
+  score: "Score",
+  locked: "Trancado",
+  unlocked: "Destrancado",
+  openmp_server: "Servidor open.mp",
+  name: "Nome",
+  ping: "Ping",
+  mode: "Modo",
+  filter_servers: "Filtrar servidores",
+  search_for_server_hostname_mode: "Procurar pelo nome/modo do servidor",
+  clear_recently_joined_list: "Limpar lista de servidores recentes",
+  refresh_servers: "Atualizar lista de servidores",
+  play: "Jogar",
+  remove_selected_server_from_favorites:
+    "Remover servidor selecionado dos favoritos",
+  add_selected_server_to_favorites: "Adicionar servidor selecionado aos favoritos",
+  add_server: "Adicionar Servidor",
+  hide_player_and_rule_list: "Ocultar lista de jogadores e regras",
+  show_player_and_rule_list: "Mostrar lista de jogadores e regras",
+  copy_server_info: "Copiar informações do servidor",
+  settings_gta_path_input_label:
+    "GTA: San Andreas caminho (onde o SA-MP está instalado)",
+  browse: "Selecionar",
+  settings_import_nickname_gta_path_from_samp:
+    "Importar apelido e caminho gtasa das configurações do SA-MP",
+  settings_import_samp_favorite_list: "Importar lista de favoritos do SA-MP",
+  settings_reset_application_data:
+    "Redefinir dados do aplicativo (limpar configurações e listas)",
+  settings_new_update_available: "⚠ Nova atualização disponível. Clique para fazer o download! ⚠",
```

Comment:
Change this part to "⚠ Nova atualização disponível. Clique aqui para baixar a atualização! ⚠"